test/parallel/test-process-redirect-warnings-env.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+// Tests the NODE_REDIRECT_WARNINGS environment variable by spawning
+// a new child node process that emits a warning into a temporary
+// warnings file. Once the process completes, the warning file is
+// opened and the contents are validated
+
+const common = require('../common');
+const fs = require('fs');
+const fork = require('child_process').fork;
+const path = require('path');
+const assert = require('assert');
+
+common.refreshTmpDir();
+
+const warnmod = require.resolve(common.fixturesDir + '/warnings.js');
+const warnpath = path.join(common.tmpDir, 'warnings.txt');
+
+fork(warnmod, {env: {NODE_REDIRECT_WARNINGS: warnpath}})
+  .on('exit', common.mustCall(() => {
+    fs.stat(warnpath, common.mustCall((err, stat) => {
```

Comment:
(By the way, these are nits. I'll go back and edit them to indicate that. You can ignore them if you choose. It can always be refactored by someone else in a subsequent PR if it bothers anyone enough. But, you know, if you like to tie up loose ends, there's a possible loose end to tie.)